packages/vkui/src/components/Group/Group.tsx

```diff
@@ -1,159 +1,34 @@
-   * По умолчанию режим отображения зависит от `sizeX` (`mode=card` при `sizeX=REGULAR` и `mode=plain` при `sizeX=COMPACT`)
-   * В модальных окнах по умолчанию `plain`.
-   */
-  mode?: 'plain' | 'card';
-  /**
-   * Отвечает за отступы вокруг контента в режиме `card`.
-   */
-  padding?: 's' | 'm';
 }
 
-const warn = warnOnce('Group');
 /**
  * @see https://vkcom.github.io/VKUI/#/Group
  */
-export const Group = ({
-  header,
-  description,
-  children,
-  separator = 'auto',
-  mode: modeProps,
-  padding = 'm',
-  tabIndex: tabIndexProp,
-  ...restProps
-}: GroupProps): React.ReactNode => {
-  const { isInsideModal } = React.useContext(ModalRootContext);
-  const { sizeX = 'none' } = useAdaptivity();
-
-  const mode = useGroupMode(modeProps, sizeX, isInsideModal);
-
-  const isTabPanel = restProps.role === 'tabpanel';
-
-  if (
-    process.env.NODE_ENV === 'development' &&
-    isTabPanel &&
-    (!restProps['aria-controls'] || !restProps['id'])
-  ) {
-    warn(
-      'При использовании роли "tabpanel" необходимо задать значение свойств "aria-controls" и "id"',
-    );
-  }
-
-  const tabIndex = isTabPanel && tabIndexProp === undefined ? 0 : tabIndexProp;
-
-  let siblingSeparatorElement: React.ReactNode = null;
-  switch (separator) {
-    case 'auto':
-      siblingSeparatorElement = <div className={styles['Group__separator-sibling']} />;
-      break;
-    case 'show':
-      siblingSeparatorElement = (
-        <div
-          className={classNames(
-            styles['Group__separator-sibling'],
-            mode === 'plain' || mode === 'none'
-              ? styles['Group__separator-sibling--forced']
-              : undefined,
-          )}
-        />
-      );
-      break;
-    case 'hide':
-      break;
-  }
-
+export const Group: React.FC<GroupProps> & {
+  Container: typeof GroupContainer;
+  Header: typeof GroupHeader;
+  Description: typeof GroupDescription;
+} = ({ header, description, children, ...restProps }: GroupProps): React.ReactNode => {
   return (
-    <>
-      <RootComponent
-        Component="section"
-        {...restProps}
-        tabIndex={tabIndex}
-        baseClassName={classNames(
-          'vkuiInternalGroup',
-          styles['Group'],
-          sizeXClassNames[sizeX],
-          mode === 'plain' && isInsideModal && styles['Group--mode-plain-inside-modal'],
-          stylesMode[mode],
-          stylesPadding[padding],
-        )}
-      >
-        {hasReactNode(header) && <div className={styles['Group__header']}>{header}</div>}
-        {children}
-        {hasReactNode(description) && (
-          <Footnote className={styles['Group__description']}>{description}</Footnote>
-        )}
-      </RootComponent>
-      {siblingSeparatorElement}
-    </>
+    <GroupContainer {...restProps}>
+      {hasReactNode(header) && <GroupHeader>{header}</GroupHeader>}
+      {children}
+      {hasReactNode(description) && <GroupDescription>{description}</GroupDescription>}
+    </GroupContainer>
   );
 };
+
```

Comment:
пропустили

```suggestion
Group.displayName = 'Group';
```

а то вот так получается:
<img width="320" alt="image" src="https://github.com/user-attachments/assets/ddddfa52-8c91-402c-a8c6-5551b9385f8a">